atom/browser/javascript_environment.cc

```diff
@@ -27,8 +27,16 @@ bool JavascriptEnvironment::Initialize() {
     const char expose_debug_as[] = "--expose_debug_as=v8debug";
     v8::V8::SetFlagsFromString(expose_debug_as, sizeof(expose_debug_as) - 1);
   }
+
+  if (cmd->HasSwitch("js-flags")) {
+    const char *js_flags_value =
+      (cmd->GetSwitchValueASCII("js-flags")).c_str();
```

Comment:
Sorry, I haven't noticed before (in my previous line comment), that js-flags is used twice in the code. When you use a string value more than once, it's time to put it in a variable, so it will be easier to maintain.
In this case I would add this declaration before line 31:
```c++
const char js_flags_switch[] = "js_flags";
```
And then use it in 31 and 33.